pkgs/by-name/fi/firefoxpwa/package.nix

```
@@ -1,5 +1,6 @@
 {
   extraLibs ? [ ],
+  runtime ? firefox-unwrapped,
```

```suggestion
  firefoxRuntime ? firefox-unwrapped,
```

Some `runtime` package might pop up in some user config